src/site/topics/java-vs-python/java-vs-python.rst

```diff
@@ -204,20 +227,22 @@ Constants
                 final double SALES_TAX = 1.15;
 
 
-* We can change constant's values however we want while writing the code
-* The point is that they will **not** change once set
+* Remember, it is possible to change a constant's value before runtime
+* The point is that they will not change at runtime
 
 
 Arrays
 ======
 
 * Java does not come with lists ready to go like Python does
-    * Or, at least, we don't get them *as a language primitive*
+
+    * They are not provided as a *language primitive*
+
 * Java does however have *arrays*, which are similar-ish to lists
+
     * They store data in a sequential linear collection
     * They have a **fixed** size
-    * They have fewer built in functions
-
+    * They have fewer built in methods
```

Comment:
Arrays don't have _methods_ -- they are just data. _Functions_ is actually right here, `operator[]` and `length` are functiony enough (ignoring the fact `operator[]` is mutable).